quay-enterprise/prometheus.md

```diff
@@ -0,0 +1,19 @@
+# Prometheus metrics under Quay Enterprise
+
+Quay Enterprise exports a [Prometheus](https://prometheus.io/)-compatible endpoint on each instance to allow for easy monitoring and alerting.
+
+## Exposing the prometheus endpoint
+
+The Prometheus-compatible endpoint on the Quay Enterprise instance can be found at port `9092`. Simply add `-p 9092:9092` to the `docker run` command (or expose the port via the `Pod` configuration in Kubernetes).
```

Comment:
It's unlikely to matter, but generally, we have [this list of default ports](https://github.com/prometheus/prometheus/wiki/Default-port-allocations) for Prometheus. Not that `9092` is explicitly left out because it's used by Kafka.

If `9092` fits in with your other ports nicely, just keep it. But if it's picked because Prometheus runs on `9090` by default, consider something different.